### PR TITLE
[script.module.pyrollbar] 0.15.2

### PR DESCRIPTION
**Pull request by LS80**
### Description

A Kodi `matrix` compatible version of `script.module.pyrollbar` (currently only on `gotham` branch).

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.pyrollbar/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyrollbar"
+       name="pyrollbar"
+       version="0.15.2"
+       provider-name="rollbar">
```
Comment (enen92) [inline script.module.pyrollbar/addon.xml:5]: This looks like a modified version of the module. Can you add yourself as an author?

Comment (LS80) [reply script.module.pyrollbar/addon.xml]: Yes it is a straight copy of the pyrollbar package plus an additional module to simplify its use within Kodi. I've added my name as an additional author.

File: script.module.pyrollbar/addon.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <import addon="script.module.six" version="1.15.0" optional="false"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Error tracking and logging from Python to Rollbar</summary>
+    <description lang="en_GB">Python notifier for reporting exceptions, errors, and log messages to Rollbar.</description>
+    <platform>all</platform>
+    <license>MIT</license>
+    <source>https://github.com/rollbar/pyrollbar</source>
```
Comment (enen92) [inline script.module.pyrollbar/addon.xml:18]: please change this to your upstream (as the module is patched). After that it's good to go.

Comment (LS80) [reply script.module.pyrollbar/addon.xml]: Done 👍

**Review by enen92 — APPROVED** [review]

**Pull request merged**